queries/latex/highlights.scm

```diff
@@ -1,17 +1,19 @@
 ; General syntax
-(command_name) @function
+(command_name) @function @nospell
```

Comment:
Shouldn't nospell be the default?

Reply:
But when some command is called inside a sentence, its parent node can be `(text)` -> `@spell`. Anyway, it was there before, I just moved it to one capture.